MAINTAINERS.md

```diff
@@ -0,0 +1,16 @@
+## Maintainers
+
+### Active Maintainers
+
+| Name | GitHub | email |
+|---|---|---|
+| Bruno Vavala | bvavala | bruno.vavala@intel.com |
+| Marcus Brandenburger | mbrandenburger | bur@zurich.ibm.com |
+
+### Retired Maintainers
```

Comment:
@g2flyer have you retired? or would you prefer _emeritus_?

Reply:
I would prefer him as active :D

Reply:
practically speaking, i certainly have no resources which would justify being called maintainer (i.e., short of some discussions, e.g., on how implemention the docker-related TODOs in the original external builder might solve some of your k8sn issues, i have no cycles to work on FPC).   So _emeritus_ probably is an accurate description. Of course, if you prefer, you could put also a _on hiatus_ :-)   You could also just leave me off this file (but at least as long as don't also kick me from the list in the README.md ;-)